storybook/stories/examples/cards.js

```diff
@@ -3,7 +3,7 @@ import React from 'react'
 import { Grid, Layout, Root } from 'reflex'
 import { Box } from '../../shared'
 import styles from './layout.css'
-import { top, horizontalHalf } from '../../shared/marginTypes'
```

Comment:
hmm why did the screenshot pic so subtly change here? is that significant enough to look into?